client/src/Components/ProductCard.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import "./productcard.css";
+
+type ProductCardProps = {
+  name: string;
+  description: string;
+  image: string;
+  creator: string;
+  purchaseHistory: number;
+};
+
+export default function ProductCard({
+  name,
+  description,
+  image,
+  creator,
+  purchaseHistory,
```

Comment:
an alternative so you could accept fewer props is to just accept one prop (product perhaps) and then access name, description, etc... on the product object that's passed in.

Reply:
Ok, totally cool with this. Let's make sure it's our group norm :)